src/main/java/za/ac/sun/cs/adversarial/agent/NegamaxAgent.java

```diff
@@ -0,0 +1,32 @@
+package za.ac.sun.cs.adversarial.agent;
+
+import za.ac.sun.cs.adversarial.algorithm.Negamax;
+import za.ac.sun.cs.adversarial.domain.Board;
+import za.ac.sun.cs.adversarial.domain.Move;
+
+import java.util.List;
+
+public class NegamaxAgent {
```

Comment:
This is not the final Negamax agent :warning: This is just something for us to make sure nothing loops out of control and that our values make sense :+1:

Reply:
We also definitely need some kind of evaluation function :smiley: